### PR TITLE
Vetoheader

### PR DESCRIPTION
**Pull request by thattil**
gotthard2 veto header not standard

### CODE REVIEW

File: slsDetectorServers/gotthard2DetectorServer/slsDetectorFunctionList.c [hunk 1]
```diff
@@ -2274,7 +2274,7 @@ void *start_timer(void *arg) {
     int datasize = imagesize;
     int packetsize = datasize + sizeof(sls_detector_header);
     int vetodatasize = VETO_DATA_SIZE;
-    int vetopacketsize = vetodatasize + sizeof(sls_detector_header);
+    int vetopacketsize = vetodatasize + VETO_HEADER_SIZE;
```
Comment (erikfrojdh) [inline slsDetectorServers/gotthard2DetectorServer/slsDetectorFunctionList.c:2277]: Why not use sizeof(veto_header)?

File: slsReceiverSoftware/src/GeneralData.h [hunk 2]
```diff
@@ -490,6 +497,7 @@ class Gotthard2Data : public GeneralData {
         vetoDataSize = 160;
         vetoPacketSize = headerSizeinPacket + vetoDataSize;
         vetoImageSize = vetoDataSize * packetsPerFrame;
+        vetoHsize = 24;
```
Comment (erikfrojdh) [inline slsReceiverSoftware/src/GeneralData.h:500]: Same here don't we have the header somewhere so that we could use sizeof?

Comment (thattil) [reply slsReceiverSoftware/src/GeneralData.h:500]: No, its in the sever definitions.  Not accsessible here.

File: slsDetectorServers/gotthard2DetectorServer/slsDetectorFunctionList.c (hunk 1 above)
Comment (thattil) [reply slsDetectorServers/gotthard2DetectorServer/slsDetectorFunctionList.c:2277]: Yes.will update

**Pull request merged**